.github/actions/build_trixie/action.yml

```diff
@@ -0,0 +1,6 @@
+# action.yml
+name: 'build-trixie'
+description: 'Build Ezra Bible App for Debian Trixie'
+runs:
+    using: 'docker'
+    image: 'docker://toklein/ezra-bullseye:latest'
```

Comment:
Please change to docker://toklein/ezra-trixie:latest

I have just created the corresponding update in https://github.com/ezra-bible-app/ezra-bible-app-docker, which generates the docker image in that location.